Dockerfile.gpu

```diff
@@ -0,0 +1,20 @@
+FROM tensorflow/tensorflow:1.12.0-gpu-py3
+
+# nvidia-docker 1.0
+LABEL com.nvidia.volumes.needed="nvidia_driver"
+LABEL com.nvidia.cuda.version="${CUDA_VERSION}"
+
+# nvidia-container-runtime
+ENV NVIDIA_VISIBLE_DEVICES=all \
+    NVIDIA_DRIVER_CAPABILITIES=compute,utility \
+    NVIDIA_REQUIRE_CUDA="cuda>=8.0" \
+    LANG=C.UTF-8
+
+RUN apt-get update && \
+    mkdir /gpt-2 && \
+    pip3 install fire==0.1.3 regex==2017.4.5
```

Comment:
ah, I should remove tensorflow from the requirements.txt perhaps?

Reply:
^ did so, so you can install from requirements.txt

Reply:
Awesome, thanks for the change, it's kind of unfortunate that tensorflow installs don't play nice with requirements.txt files.